### DISCUSSION TITLE
notificaion show in notificaion area but not popup

### DISCUSSION THREAD

**Discussion by Nader-CS** [invertase/react-native-firebase#8202] [Q&A]
why notificaion shown in notificaion area when app is killed or in background , but popup when in foregreound

**Comment by mikehardy** [comment]
There is no useful information provided that would help anyone assist you.
Relevant information would be the JSON payload sent as an FCM, and the minimum required code in index.js and App.js that could be placed in an empty example app to reproduce the issue, along with all versions of all software in use.

https://stackoverflow.com/help/how-to-ask
https://stackoverflow.com/help/minimal-reproducible-example
